### PR TITLE
quic-tls nits

### PR DESCRIPTION
**Pull request by martinduke**
I suspect these are uncontroversial.

### CODE REVIEW

File: draft-ietf-quic-tls.md [hunk 1]
```diff
@@ -788,7 +788,7 @@ The value of initial_salt is a 20 byte sequence shown in the figure in
 hexadecimal notation. Future versions of QUIC SHOULD generate a new salt value,
 thus ensuring that the keys are different for each version of QUIC. This
 prevents a middlebox that only recognizes one version of QUIC from seeing or
-modifying the contents of handshake packets from future versions.
+modifying the contents of Initial packets from future versions.
```
Comment (marten-seemann) [inline draft-ietf-quic-tls.md]: Initial is a version-dependent concept.

Comment (martinduke) [reply draft-ietf-quic-tls.md]: but the concept of the version 1 salt is directly related to Initial packets...

Comment (marten-seemann) [reply draft-ietf-quic-tls.md]: True, but you still can't talk about "Initial packets from future versions". Future version might not even have Initial packets.

Comment (martinthomson) [reply draft-ietf-quic-tls.md]: @marten-seemann's point is right.  The intent here is to use a generic statement (packets that are part of the handshake process), as opposed to identifying a specific packet type in this version.

```suggestion
modifying the contents of packets from future versions.
```

**Comment by martinduke** [comment]
All three Mart[ie]ns now agree, no further objections allowed.

**Pull request merged**